src/panels/config/users/ha-config-users.ts

```diff
@@ -46,49 +46,68 @@ export class HaConfigUsers extends LitElement {
           width: "25%",
           direction: "asc",
           grows: true,
-          template: (name) => html`
-            ${name ||
-            this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
-          `,
-        },
-        username: {
-          title: this.hass.localize(
-            "ui.panel.config.users.picker.headers.username"
-          ),
-          sortable: true,
-          filterable: true,
-          width: "20%",
-          direction: "asc",
-          template: (username) => html`
-            ${username ||
-            this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
-          `,
-        },
-        group_ids: {
-          title: this.hass.localize(
-            "ui.panel.config.users.picker.headers.group"
-          ),
-          sortable: true,
-          filterable: true,
-          width: "20%",
-          template: (groupIds) => html`
-            ${this.hass.localize(`groups.${groupIds[0]}`)}
-          `,
+          template: (name, user: any) =>
+            narrow
+              ? html` ${name}<br />
+                  <div class="secondary">
+                    ${user.username} |
+                    ${this.hass.localize(`groups.${user.group_ids[0]}`)}
+                  </div>`
+              : html` ${name ||
+                this.hass!.localize(
+                  "ui.panel.config.users.editor.unnamed_user"
+                )}`,
         },
       };
-      if (!narrow) {
-        columns.system_generated = {
-          title: this.hass.localize(
-            "ui.panel.config.users.picker.headers.system"
-          ),
-          type: "icon",
-          sortable: true,
-          filterable: true,
-          width: "160px",
-          template: (generated) =>
-            generated ? html`<ha-icon icon="hass:check"> </ha-icon>` : "",
-        };
-      }
+
+      columns.username = {
```

Comment:
No need to do this separate, can all be done in de same declaration (line 39)

Reply:
True. With my previous commits that used some `if` branches to not show some columns, it was required, but not anymore with the `hidden:` approach. Fixed.